samples/beta/delete_dataset_test.py

```diff
@@ -27,13 +27,13 @@
 @pytest.fixture(scope="function")
 def dataset_id():
     client = automl.AutoMlClient()
-    project_location = client.location_path(PROJECT_ID, "us-central1")
+    project_location = f"projects/{PROJECT_ID}/locations/us-central1"
```

Comment:
Is there no longer a `location_path` method?

Reply:
Yes, this is one that tends to disappear when the API gets moved off of the gapic yaml. https://github.com/googleapis/googleapis/blob/2db5725bf898b544a0cf951e1694d3b0fce5eda3/google/cloud/automl/v1/automl_gapic.legacy.yaml#L35-L44

Reply:
I was under the impression that a path helper is generated if a pattern is the first listed for a resource. Is that correct? https://github.com/googleapis/googleapis/blob/master/google/cloud/automl/v1/dataset.proto#L38

Reply:
That is correct; at most one path helper is generated per resource. If there are multiple paths, then the path chosen is the first in the annotation.